SMAnalysis/schema-LD.json

```diff
@@ -0,0 +1,52 @@
+{
+  "$schema": "http://json-schema.org/schema#",
+  "schemaVersion": "0.0",
+  "$id": "https://raw.githubusercontent.com/smart-data-models/dataModel.SocialMedia/master/SMAnalysis/schema.json",
+  "title": "Social Media - SMAnalysis",
+  "description": "This entity contains a harmonised description of a generic SMAnalysis made for the Social Media domain. This entity is primarily associated with the process of analysis of Social Media applications' posts.",
+  "type": "object",
+  "allOf": [
+    {
+      "$ref": "https://smart-data-models.github.io/data-models/common-schema.json#/definitions/Location-Commons"
+    },
+    {
+      "properties": {
+	  "type": {
+          "type": "string",
+          "description": "Property. NGSI-LD Entity Type. It must be equal to SMAnalysis.",
+          "enum": [
+            "SMAnalysis"
+          ]
+        },
+	  "hasConfidence": {
+          "type": "number",
+          "description": "Property.  Model:'https://schema.org/Number'. Units:'No unit'. It represents the confidence of the analysis. All units are accepted in [CEFACT](https://www.unece.org/cefact.html) code."
+        },
+	  "analyzedAt": {
+          "type": "string",
+          "format": "date-time"
+        },
+	  "hasAnalysisValue": {

```

Comment:
duplicated

Reply:
What is duplicated?

Reply:
all. A data model has only one schema.json (not one schema.json and schema-LD.json)

Reply:
Each schema handles different payloads in my understanding, but if this is wrong, feel free to not accept the schema-LD.json files.

Reply:
I'll be glad to comment on the new schema format for validation of other payload types (not just the key values). But in the incubated repo not in the official one.

Reply:
Thank you Alberto. So, I will make a PR to upload the schema-ld and example-ld json files to incubated repo, thanks.

Reply:
Would you care to make a SocialMedia folder in incubated repo?